feature-libs/product/configuration/src/textfield-configurator/facade/configurator-textfield.service.spec.ts

```diff
@@ -2,22 +2,23 @@ import { Type } from '@angular/core';
 import { async, TestBed } from '@angular/core/testing';
 import * as ngrxStore from '@ngrx/store';
 import { Store, StoreModule } from '@ngrx/store';
-import { Observable, of } from 'rxjs';
-import { ActiveCartService } from '../../../cart/facade/active-cart.service';
-import { Cart } from '../../../model/cart.model';
-import { ConfiguratorTextfield } from '../../../model/configurator-textfield.model';
-import { GenericConfigurator } from '../../../model/generic-configurator.model';
-import { OCC_USER_ID_ANONYMOUS } from '../../../occ/utils/occ-constants';
-import { ProcessesLoaderState } from '../../../state/utils/processes-loader/processes-loader-state';
-import * as ConfiguratorActions from '../store/actions/configurator-textfield.action';
-import { StateWithConfigurationTextfield } from '../store/configuration-textfield-state';
+import {
+  ActiveCartService,
+  Cart,
+  GenericConfigurator,
+  OCC_USER_ID_ANONYMOUS,
+} from '@spartacus/core';
+import { of } from 'rxjs';
+import { ConfiguratorTextfield } from '../model/configurator-textfield.model';
+import { ConfiguratorTextfieldActions } from '../state/actions/index';
```

Comment:
Sorry, I just noticed that you are missing a _grouping_ file in your actions and selectors.
You should create the _group.ts_ file (e.g. `projects/core/src/cms/store/actions/cms-group.actions.ts`) and then have something like this in the `index.ts`:
```ts
import * as CmsActions from './cms-group.actions';
export { CmsActions };
```

Reply:
I saw this yesterday but thought it was not needed because we only have one actions file...Why is this needed?

Reply:
Still I will create those files..